app/src/main/java/com/pajato/android/gamechat/common/PlayModeManager.java

```diff
@@ -95,12 +97,14 @@ public void showPlayModeMenu(View anchorView) {
     private List<ListItem> getUserItems() {
         List<ListItem> result = new ArrayList<>();
         Account account = AccountManager.instance.getCurrentAccount();
-        if (account == null || account.joinList.size() == 0)
-            return result;
-        for (String groupKey : account.joinList)
-            for (Account member : MemberManager.instance.getMemberList(groupKey))
-                if (!member.id.equals(account.id))
-                    result.add(new ListItem(new UserItem(groupKey, member)));
+        if (!AccountManager.accountHasFriends(account))
+            // No users exist so create a header stating this.
+            result.add(new ListItem(resourceHeader, R.string.NoFriendsFound));
+        else
+            for (String groupKey : account.joinList)
+                for (Account member : MemberManager.instance.getMemberList(groupKey))
+                    if (!member.id.equals(account.id))
+                        result.add(new ListItem(new UserItem(groupKey, member)));
```

Comment:
A request:  if  you find yourself working on code that uses a helper class, like UserItem, that we are trying to get rid of, by all means, avail yourself of the opportunity to include that in your current piece of work.